content/docs/introduction/regions.md

```diff
@@ -51,6 +59,12 @@ If you are unsure of your project's region, you can find this information in the
 | Asia Pacific (Singapore) — aws-ap-southeast-1 | 54.254.50.26, 54.254.92.70, 54.255.161.23                                              |
 | Asia Pacific (Sydney) — aws-ap-southeast-2    | 13.237.134.148, 13.55.152.144, 54.153.185.87                                           |
 
+### Azure NAT Gateway IP Addresses
+
+| Region                               | NAT Gateway IP Addresses                                                               |
+| :----------------------------------- | :------------------------------------------------------------------------------------- |
+| East US 2 (Virginia) — azure-eastus2 | 23.23.0.232, 3.222.32.110, 35.168.244.148, 54.160.39.37, 54.205.208.153, 54.88.155.118 |
```

Comment:
We can update with these IPs: https://docs.neon.build/infrastructure/kubernetes/aks.html#prod

Reply:
updated. TY!